bin/aws/ssm_consul_server.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SERVERS=$(aws ec2 describe-instances --filter Name=tag:Name,Values="Consul Server")
+CONSUL_SERVER_INSTANCE_ID=$(echo "${SERVERS}" | jq -r .Reservations[0].Instances[0].InstanceId)
+
+echo "--- Instance: ${CONSUL_SERVER_INSTANCE_ID}"
+
+aws ssm start-session \
+    --target "${CONSUL_SERVER_INSTANCE_ID}" \
+    --document-name AWS-StartPortForwardingSession \
+    --parameters '{"portNumber":["8500"], "localPortNumber": ["8500"]}'
```

Comment:
Might be useful to parameterize the local port number (with a default value of 8500), for those of us that might plausibly be connecting to servers in multiple accounts.